serverlessgo/input.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"log"
+	"strconv"
+	"encoding/csv"
+)
+
+type inputEntry struct {
+	duration float64
+	status   int
+}
+
+func buildEntryArray(p string) ([]inputEntry, error) {
+	f, err := os.Open(p)
+	if err != nil {
+		return nil, err
```

Comment:
fmt.Errorf("Error opening the file (%s), %q", p, err)

Reply:
Done